src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [filmTitle, searchFilm] = useState('');
```

Comment:
```suggestion
  const [searchQuery , setSearchQuery] = useState('');
```
The name is a bit confusing as it's the value which the user types in input, not necessarily the film's title